packages/mobile/src/app/App.tsx

```diff
@@ -70,7 +70,7 @@ export class App extends React.Component<Props> {
     // Handles opening Clevertap deeplinks when app is closed / in background
     CleverTap.getInitialUrl(async (err: any, url) => {
       if (err) {
-        Logger.error('App/componentDidMount', 'App CleverTap Deeplink on Load', err)
+        Logger.debug('App/componentDidMount', 'App CleverTap Deeplink on Load', err)
```

Comment:
i don't know anything about this code, but downgrading to `debug` from `error` seems drastic and I would have expected a `warn` instead.  Any reason you moved it to `debug`?

Reply:
@lisaqyli we have a remote config variable for exact errors that we want to stop reporting on, it's called `sentryNetworkErrors` - are these errors something that we should add to this variable, rather than make changes to the code? the benefit of using `sentryNetworkErrors` is that we can exclude specific error values (rather than exclude specific parts of the code where we catch errors, where the error can be any value)

Reply:
Could we filter out the exact error instead?